tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -1992,7 +2009,7 @@ private void doAutoImportOldWinningVotes(SurveyJSONWrapper r, User user, SurveyM
             return;
         }
         alreadyAutoImportedVotes(user.id, "set");
-
+        CookieSession.sm.getSTFactory().setupDB();
```

Comment:
For the record could just skip if not existing. But ok.

Reply:
I don't think "could just skip": if the new table doesn't exist yet, the old votes won't get imported unless the table gets created here.